src/logo-system/logo-elements/index.md

```diff
@@ -9,23 +9,23 @@ Our wordmark has been redrawn, elevating the dot into a position that signifies
 
 As our primary identifier, the GOV.UK wordmark should be used in all applications of the logo.
 
-The exception to this rule is the GOV.UK website. See the Web Logo section (3.3) for more details.
+The exception to this rule is the GOV.UK website. See the [Web – Logo system page](/logo-system/web/).
 
-![TODO](./wordmark.svg)
+![](./wordmark.svg)
```

Comment:
I'd generally agree that these are decorative. But because the wordmark is the actual subject of this paragraph and is re-used throughout the section, I think it should be described at least once. And here is probably the best place for that. (Or alternatively on /logo-system/?).

Without seeing it, you cannot know that it contains the text "GOV.UK", for example.